src/systemcmds/param/param.cpp

```diff
@@ -141,6 +143,8 @@ Change the airframe and make sure the airframe's default parameters are loaded:
 	PRINT_MODULE_USAGE_PARAM_FLAG('q', "quiet mode, print only param value (name needs to be exact)", true);
 	PRINT_MODULE_USAGE_ARG("<filter>", "Filter by param name (wildcard at end allowed, eg. sys_*)", true);
 
+	PRINT_MODULE_USAGE_COMMAND_DESCR("show-for-airframe", "Show changed params for airframe config");
```

Comment:
@bkueng Why not `param show` and a flag? ie `param show -f`

Reply:
It would become a bit convoluted with the other flags (e.g. `-a` and `-q` flags would not make sense in presence of `-f`). But if you prefer, I don't mind much going with the flag.

Reply:
@bkueng I have no strong opinion/do as you will. Just making the point.

What does "Show changed params for airframe config" mean? Show just the params listed in the airframe init file that have been changed from the defaults? I.e. for example, if using this for HIL, the params listed here that have also changed? https://github.com/PX4/Firmware/blob/master/ROMFS/px4fmu_common/init.d/airframes/1000_rc_fw_easystar.hil (say) 

Or does it further include the included stuff like "sh /etc/init.d/rc.fw_defaults"

Reply:
yes it includes `sh /etc/init.d/rc.fw_defaults` - definitely not perfect.

Reply:
Good by me. We should provide this additional detail in https://dev.px4.io/master/en/advanced/parameters_and_configurations.html once you've got it in :-)